policyengine_us/parameters/gov/states/nc/tax/income/credits/ctc/head_of_household.yaml

```
@@ -0,0 +1,30 @@
+description: North Carolina allows for the following progressive child tax credit for head of household filers.
```

```suggestion
description: North Carolina provides for the following child tax credit amount for head of household filers, depending on their federal adjusted gross income.
```
similar language in all yaml